src/schema/__tests__/FroidSchema.test.ts

```diff
@@ -549,7 +549,7 @@ describe('FroidSchema class', () => {
           bookId: String!
         }
 
-        type Magazine implements Node @key(fields: "magazineId publisher { __typename address { __typename country } }") {
+        type Magazine implements Node @key(fields: "magazineId publisher { __typename address { __typename country postalCode } }") {
```

Comment:
This key now correctly inherits the `postalCode` field in its key. This was a previously unidentified gap.

You may notice that the `Book` entity is _not_ getting the `country` field. This is because `Author.address` is external and, therefore, it fields won't be resolved through the `Book`.